packages/desktop-electron/bin/update-client.js

```diff
@@ -0,0 +1,18 @@
+#!usr/bin/env node
+const { fsUtil, Path } = require('../../../bin/utils');
+
+const ROOT = process.cwd();
```

Comment:
How would this work if being run from the root of the whole repo? I believe `` `dirname $0`/..`` will return `/path/to/actual/packages/desktop-electron` no matter what the cwd is. (maybe it would be worth adding `projectRoot`, and maybe `packageRoot('desktop-electron')` too, as an export from `bin/utils`)?